package.json

```diff
@@ -43,7 +43,6 @@
     "@storybook/addon-options": "^3.4.11",
     "@storybook/addon-viewport": "^3.4.11",
     "@storybook/react": "4.0.0-alpha.8",
-    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Just tried locally and this doesn't appear to be needed - perhaps because we have that plugin-transform-modules-commonjs plugin for Jest now 🤔

Reply:
Nope - wait - after running `npm ci` and then `npm test`, it fails "cannot find babel-core" so we do need it.

Reply:
What is the reason for using the alpha of @storybook/react as opposed to something more recent - `"^4.0.12"`?